doc/source/docs/api/testing.rst

```diff
@@ -0,0 +1,14 @@
+=======
+Testing
+=======
+.. currentmodule:: geopandas
+
+GeoPandas includes specific functions to test its objects.
+
+.. autosummary::
+   :toctree: api/
+
+   testing.geom_equals
+   testing.geom_almost_equals
```

Comment:
I am not sure we should advertize those two (I know they are strictly speaking public .. ;), but still, people should in general use the assert_ methods for testing)